### PR TITLE
Build kolibri windows installer using docker.

### PR DESCRIPTION
**Pull request by mrpau-richard**
## Summary

I created a script to build kolibri windows installer at buildkite.

Here's the [sample build plan](https://buildkite.com/learningequality/mrpau-kolibri-installer/builds/1) for this branch at buildkite.

/cc @aronasorman @cpauya @radinamatic

### CODE REVIEW

**Comment by mrpau-richard** [comment]
Hi @radinamatic can you test the kolibri windows installer build by this PR [here](https://buildkite.com/learningequality/kolibri/builds/421#3992ae66-f4a6-49e6-8528-a8ac43bf664b).

**Comment by radinamatic** [comment]
@mrpau-richard Installer apparently works a-Ok, just two comments:

1. Any reason the Python version installed is `2.7.10` instead of `2.7.12` used on KA Lite?
virtualbox_windows 7_12_04_2017_13_18_00

2. Seems that we are still in 2015... 😉 
virtualbox_windows 7_12_04_2017_13_18_52

**Comment by mrpau-richard** [comment]
@radinamatic I haven't tested it yet in Python `2.7.12`. We also need to merge this [PR](https://github.com/learningequality/kolibri-installers/pull/1) to update the year of the LICENSE file.

**Comment by radinamatic** [comment]
Ok, understood, looks good to me then! 👍

File: windows_installer_docker_build/export.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Move built kolbiri windows installer in installer path.
+mv /kolibri/kolibri-installers/windows/*.exe /installer/
```
Comment (aronasorman) [inline windows_installer_docker_build/export.sh]: Can we move this to inside the `.buildkite` folder as well?

Comment (mrpau-richard) [reply windows_installer_docker_build/export.sh]: @aronasorman I already moved it at docker file.

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -11,3 +11,8 @@ steps:
 
   - label: Build the pex file
     command: mkdir -p dist && .buildkite/build_pex.sh
+
+  - wait
```
Comment (aronasorman) [inline .buildkite/pipeline.yml]: Oh, and no need for this wait clause. By removing it we can build concurrently with the pex file.

Comment (mrpau-richard) [reply .buildkite/pipeline.yml]: I already removed it @aron. And its build fine at [buildkite](https://buildkite.com/learningequality/kolibri/builds/431#64efe103-fea3-49be-9b20-6c3dda765d91)

**Pull request merged**